modules/ztp-configuring-ptp-fast-events-amqp.adoc

```diff
@@ -20,67 +21,54 @@ include::snippets/ptp-amq-interconnect-eol.adoc[leveloffset=+1]
 
 .Procedure
 
-. Add the following YAML into `.spec.sourceFiles` in the `common-ranGen.yaml` file to configure the AMQP Operator:
+. Add the following YAML into `{rangen-yaml-path}` in the `{policy-prefix}common-ranGen.yaml` file to configure the AMQP Operator:
```

Comment:
Verify this and any other policy prefix uses. I'm seeing rendered names that look like they might want an extra dash, but I am quite probably wrong--this isn't my lane.

Reply:
These should all render like the files in https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example/acmpolicygenerator

I don't see any errors in the output though?